go/pkg/pwcompose/compose.go

```diff
@@ -172,11 +172,11 @@ func Prepare(challengeDir string, prefix string, noPush bool, version string, lo
 	// print yaml
 	finalData, err := yaml.Marshal(&composeStruct)
 	if err != nil {
-		return errcode.ErrComposeMarshalConfig.Wrap(err)
+		return "", errcode.ErrComposeMarshalConfig.Wrap(err)
 	}
 	fmt.Println(string(finalData))
```

Comment:
since you return finalData, you should not print it here, but from the main

Reply:
noice